client/signup/config/flows-pure.js

```diff
@@ -398,12 +398,22 @@ export function generateFlows( {
 				'difm-site-picker',
 				'site-info-collection',
 				'difm-design-setup-site',
+				'domains',
 			],
 			destination: getDIFMSignupDestination,
 			description: 'A flow for DIFM Lite leads',
 			excludeFromManageSiteFlows: true,
 			lastModified: '2021-09-30',
 		},
+
+		{
+			name: 'site-content-collection',
+			steps: [ 'website-content' ],
+			destination: null,
```

Comment:
I think we'll need to add a destination here. Maybe just a function redirecting to `/home/${siteSlug}`